Packs/Respond/pack_metadata.json

```diff
@@ -1,11 +1,11 @@
 {
-    "name": "Respond Analyst",
-    "description": "Respond Analyst Integration",
+    "name": "Mandiant Automated Defense",
+    "description": "Mandiant Automated Defense Pack. See https://www.mandiant.com/advantage/automated-defense for more information.",
```

Comment:
Let's remove the link as it is not something you would expect to appear in a pack marketplace card.